tests/DriverManagerTest.php

```diff
@@ -149,6 +149,51 @@ public function testDatabaseUrlPrimaryReplica(): void
         );
     }
 
+    public function testDatabaseUrlPrimaryReplicaWithServerVersionInUrl(): void
+    {
+        $options = [
+            'driver' => 'pdo_mysql',
+            'primary' => ['url' => 'mysql://foo:bar@localhost:11211/baz?serverVersion=11.2'],
+            'replica' => [
+                'replica1' => ['url' => 'mysql://foo:bar@localhost:11211/baz_replica?serverVersion=11.2'],
```

Comment:
If the server version is specified both in the primary and replica URLs, which of them is expected to be passed to the replica driver?

Reply:
Database platform seem to be shared between primary and replica. I assume that implementing ability to set different versions between them would require connection logic to be rewritten.